<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dictionary</class>
 <widget class="QWidget" name="Dictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>483</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>483</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dictionary</string>
  </property>
  <widget class="QListWidget" name="lwDictionary">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDictionary">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dictionary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Word</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbHard">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">rbGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rbEasy">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">rbGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rbMedium">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">rbGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rbGroup"/>
 </buttongroups>
</ui>
